build/packages/rpm/server/SPECS/grafana.spec

```diff
@@ -2,8 +2,8 @@
 %global commit          33423d34f211ce1ce5ce0a265a38f0709ec44360
 %global shortcommit     %(c=%{commit}; echo ${c:0:7})
 %define build_timestamp %(date -u +"%y%m%d%H%M")
-%define release         98
-%define grafana_version 9.2.13
+%define release         99
```

Comment:
@filipmikes1 I'm afraid we need to update the commit on line 2 as well :)

Reply:
Hmm, right, where's that hash coming from?

Reply:
From the upstream repo. You can take it from the relevant [tag](https://github.com/grafana/grafana/releases/tag/v9.2.18), that appears be [2a6a04d0eb8bbe800eca86d8e7d3018a69a4daa6](https://github.com/grafana/grafana/commit/2a6a04d0eb8bbe800eca86d8e7d3018a69a4daa6).

Reply:
Sorry, I was wrong :| Take a commit hash from our own repo - https://github.com/percona-platform/grafana

Reply:
@atymchuk hash updated with squash commit